src/containers/pages/MDAPoint/ClientUpload/index.tsx

```diff
@@ -70,7 +72,7 @@ export const ClientUpload = (props: any) => {
                   <div style={{ display: 'inline-block', width: '24rem' }}>
                     <Field
                       required={true}
-                      component={JurisdictionSelect}
+                      component={JurisdictionSelectHOF}
```

Comment:
Why this change?

Reply:
@moshthepitt JurisdictionSelectHOF returns the JurisdictionSelect with the flag that allows us to drill down to location level. It is inspired by this https://jaredpalmer.com/formik/docs/api/field#example